### DISCUSSION TITLE
v2.0.27

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#288] [Announcements]
## Changes

- Few more tweaks @DTTerastar (#287)
- Fix all issues found debugging @DTTerastar (#285)
- Reduce stack size @DTTerastar (#284)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.27'>v2.0.27</a>.</em>

**Comment by erwinSmith757** [comment]
this version on a D1 mini... reports once... then nothing,,,,,,

should i rather use a different device? due this project in principal is amazing !!!

**Comment by tasi333** [comment]
@DTTerastar Just installed 2.0.28 on a VROOM32 device, seems stable so far, no reboots.

**Reply by DTTerastar** [reply]
My testing has uptimes in hours.  Not great, but better than 30 seconds :)

**Reply by HeavyHawkZero** [reply]
Thanks for the great and fast work. I also upgraded to 2.0.28. No reboots so far but unfortunately, my devices have stopped to report devices alltogether. I filed a bug report for that.
Again, thanks for all the work you do for this project. Very much appreciated.
